agent/src/com/jonnyzzz/teamcity/virtual/run/VMRunnerContext.java

```diff
@@ -16,6 +16,7 @@
 
 package com.jonnyzzz.teamcity.virtual.run;
 
+import com.intellij.openapi.util.SystemInfo;
```

Comment:
Same here. A bit correct is to use AgentConfiguration to get information on whether it's Linux or Windows. SystemInfo is meanwhile in the API, but not the best way to use